nuxt.config.ts

```diff
@@ -14,26 +14,89 @@ export default defineNuxtConfig({
       },
     },
   },
-  plugins: [
-    '~/plugins/pinia-plugin-persist.client'
-  ],
+  plugins: ["~/plugins/pinia-plugin-persist.client"],
   modules: [
+    "@nuxtjs/i18n",
     [
-      '@pinia/nuxt',
+      "@pinia/nuxt",
       {
-        autoImports: [
-          'defineStore',
-          ['defineStore', 'definePiniaStore']
-        ],
+        autoImports: ["defineStore", ["defineStore", "definePiniaStore"]],
       },
     ],
   ],
   imports: {
-    dirs: ['stores'],
+    dirs: ["stores"],
   },
   runtimeConfig: {
     public: {
       baseUrl: process.env.API_BASE_URL,
-    }
+    },
+  },
+  i18n: {
+    locales: [
+      { code: "en", name: "English", iso: "en-US", file: "en-US.js" },
+      { code: "ar", name: "Arabic", iso: "ar", file: "ar.js" },
+      { code: "bn", name: "Bangla", iso: "bn-BD", file: "bn-BD.js" },
+    ],
+    defaultLocale: "en",
+    strategy: "prefix_except_default",
+    vueI18n: {
+      legacy: false,
+      locale: "en",
+      fallbackLocale: "en",
+      messages: {
```

Comment:
is it possible to create separate files for messages according to language? Then will include/imports them in `nuxt.config` otherwise the size of `nuxt.config` will be increased.